ampersand-select-view.js

```diff
@@ -243,6 +257,20 @@ SelectView.prototype.getOptionValue = function (option) {
     return option;
 };
 
+SelectView.prototype.getOptionDisabled = function (option) {
+    if (Array.isArray(option)) return;
+
+    if (this.options.isCollection) {
+        if (this.disabledAttribute && option[this.disabledAttribute]) {
```

Comment:
This can be reduced to

```js
if (this.disabledAttribute) return option[this.disabledAttribute];
```

As it'll return exactly same logical thing.